tests/test_cli.py

```diff
@@ -277,9 +277,9 @@ def test_cli_searchvar(self):
 
     def test_cli_inventory(self):
         """
-        run $ kapitan inventory -t minikube-es -F -p cluster
+        run $ kapitan inventory --target-name minikube-es --flat --pattern cluster
```

Comment:
Why are you expanding the arguments?

Reply:
Just for the same reason mentioned above. I have checked in the current version they only support shorthand notation for bool types but to maintain uniformity I updated all flags.

Reply:
We can't remove the short notation as we and other companies mostly use that.

Reply:
I agree as well and shorthand notation is indeed mostly used.
Once the python-fire's new version is released then even with shorthand notation searchvar, inventory, init and eval will have no problem as each of them have unique first letter in their arguments.
compile, lint and secrets have some conflicts which would be pointed out by fire if someone tries that.